libs/application/template-api-modules/src/lib/modules/templates/health-insurance/health-insurance.service.ts

```diff
@@ -0,0 +1,141 @@
+import { Injectable } from '@nestjs/common'
+import { logger } from '@island.is/logging'
+
+import { SharedTemplateApiService } from '../../shared'
+import { TemplateApiModuleActionProps } from '../../../types'
+import { transformApplicationToHealthInsuranceDTO } from './health-insurance.utils'
+
+@Injectable()
+export class HealthInsuranceService {
+  constructor(
+    private readonly sharedTemplateAPIService: SharedTemplateApiService,
+  ) {}
+
+  async sendApplyHealthInsuranceApplication({
+    application,
+    authorization,
+  }: TemplateApiModuleActionProps) {
+    try {
+      logger.info(
+        `Start send Health Insurance application for ${application.id}`,
+      )
+      const vistaSkjal = transformApplicationToHealthInsuranceDTO(application)
+      logger.info(`Finished transform Application to Health Insurance DTO`)
+
+      logger.info(`Start query`)
+      const attachmentsStr = `["${vistaSkjal.attachmentsFileNames?.join(
+        '","',
+      )}"]`
+
+      const query = `mutation {
+        healthInsuranceApplyInsurance(
+        inputs: {
+          applicationNumber: "${vistaSkjal.applicationNumber}",
+          applicationDate: "${vistaSkjal.applicationDate}",
+          nationalId: "${vistaSkjal.nationalId}",
```

Comment:
We should definitely use [graphql variables](https://graphql.org/graphql-js/passing-arguments/) instead of string interpolation in queries (which makes us vulnerable to GraphQL injection attacks). To be fair, this feature is missing from the makeGraphqlQuery.

Reply:
Move codes under template is next step